cardano/backends/walletrest/__init__.py

```diff
@@ -45,25 +47,29 @@ class WalletREST(object):
         },
     }
 
-    def __init__(self, protocol="http", host="localhost", port=8090):
+    def __init__(self, protocol="http", host="localhost", port=8090, verify=None, cert=None):
         self.base_url = "{protocol}://{host}:{port}/v2/".format(
             protocol=protocol, host=host, port=port
         )
+        self.verify=verify
+        self.cert=cert
         _log.debug("WalletREST backend url: {:s}".format(self.base_url))
 
-    def raw_request(self, method, path, params=None):
+    def raw_request(self, method, path, params=None, verify=None, cert=None):
```

Comment:
It seems this method doesn't use the new keywords

Reply:
Yikes, seems like I need to get some sleep.
I'll take a look at this (and other comments) tomorrow with fresh eyes and will re-submit.
Thanks for fast review!